### ISSUE TITLE
Check packages generated have the correct versions

### ISSUE THREAD

**Issue by fgsch** [varnishcache/pkg-varnish-cache#11]
I found files in repo.varnish-cache.org with version-version on top of the regular version, e.g. varnish-doc_4.1.1-4.1.1~jessie_all besides varnish-doc_4.1.1-1~jessie_all.

This needs to be checked before 4.1.2.

**Comment by lkarsten** [comment]
This version string is based on the version string from configure. On debs part of it is overridable at build time in the Jenkins UI.

Overrides are UPSTREAMVERSION and DEBVERSION. The first one isn't used in VC4.0/VC4.1 jobs, but since we'd like to keep the 3.0 jobs semi-working still, we won't fix that just yet. DEBVERSION allows use to override the x in 4.1.1-x if we need that.

If DEBVERSION isn't set, we get the daily builds format shown here:
```
-rw-rw-r--  1 lkarsten lkarsten  532144 feb.  18 11:18 varnish_4.1.2-0beta1+daily+20160218.111550~jessie_amd64.deb
-rw-rw-r--  1 lkarsten lkarsten  374114 feb.  18 11:18 varnish-doc_4.1.2-0beta1+daily+20160218.111550~jessie_all.deb
```

What I think happened here, is that I put "4.1.1" into DEBVERSION (instead of UPSTREAMVERSION) while doing the manual version override. Because of the varnishncsa.service sighup issues in 4.1.1 packaging, I rebuilt these a couple of times in a hurry. (also why it is at 4.1.1-4 now, or something like that.)

Summing up: most likely this was me using the wrong field. I think I'll manage not to do in the future, and when we can kill off the 3.0 stuff (or care enough to rewrite those jobs) we'll remove the manual input step that made this possible.

I'm closing this issue for now.
